internal/bminventory/inventory_test.go

```diff
@@ -7506,96 +7506,56 @@ var _ = Describe("KubeConfig download", func() {
 
 var _ = Describe("DownloadMinimalInitrd", func() {
 	var (
-		bm        *bareMetalInventory
-		cfg       Config
-		db        *gorm.DB
-		ctx       = context.Background()
-		clusterID strfmt.UUID
-		infraEnv  common.InfraEnv
-		dbName    string
+		bm     *bareMetalInventory
+		cfg    Config
+		db     *gorm.DB
+		ctx    = context.Background()
+		id     strfmt.UUID
+		dbName string
+		httpProxy = "http://10.10.1.1:3128"
+		httpsProxy = "https://10.10.1.1:3128"
+		noProxy = "quay.io"
```

Comment:
Lack of alignment here might fail the linter. I'd be sure to `skipper make validate` before making this a non-draft.